commands/files/serialfile.go

```diff
@@ -68,6 +70,15 @@ func (f *serialFile) NextFile() (File, error) {
 	stat := f.files[0]
 	f.files = f.files[1:]
 
+	for !f.hidden && strings.HasPrefix(stat.Name(), ".") {
```

Comment:
Probably best to rename `.hidden` to something like `.addHidden` or `.ignoreHidden`